### PR TITLE
[KOA-5480] Update buttons to semantic tokens and adopt new colour palette

### PR DESCRIPTION
**Pull request by anambl**
Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+- `@skyscanner/backpack-web`:
+  - Update `bpk-scrim-utils` component to semantic colours
+  - Deprecated `dark` property in `bpk-scrim-utils` component as we only support one scrim style and this property could never be applied.
 
 **Fixed:**
 
-- bpk-component-accordion:
-    - Remove redundant `aria-labeledby` tags in accordion item
+- `@skyscanner/backpack-web`:
+  - Remove redundant `aria-labeledby` tags in accordion item in accordion component
```
Comment (olliecurtis) [inline UNRELEASED.md:16]: If we are changing this to be the single package, then what we should do is above all the changes/headings we should just put `@skyscanner/backpack-web` at the top

So like

```
`@skyscanner/backpack-web`

  **Breaking:**
    - Switch all components to use new colour palette.
   
    `BpkButton`
      - Update buttons to semantic colour tokens.
    
  **Changed:**
    `scrim-utils`
       - Update `bpk-scrim-utils` component to semantic colours
       - Deprecated `dark` property in `bpk-scrim-utils` component as we only support one scrim style and this property could never be applied.

  **Fixed:**
    `BpkAccordion`
      - Remove redundant `aria-labeledby` tags in accordion item in accordion component

```

**Review by olliecurtis — CHANGES_REQUESTED** [review]
Just a small formatting query now we are only shipping single package

File: UNRELEASED.md (hunk 1 above)
Comment (anambl) [reply UNRELEASED.md:16]: I think it's ok to add `backpack-web` at the top, but we have a few components that are exporting multiple components one of them being the button which will also export `BpkButtonPrimary` etc. so I don't think that `BpkButton` works. I do think though that we need something to make it very easy to understand what component it refers to. I'm thinking we could still add `bpk-component-button` just won't add a version next to it 🤔

Comment (olliecurtis) [reply UNRELEASED.md:16]: Sounds good to me! The other way we could do it is whatever the component name is after the `bpk-compoment-` similar to iOS and Android.

Either works but we just need to move to only noting the single package once

Comment (anambl) [reply UNRELEASED.md:16]: Good point, I see iOS is using `Backpack/Button` we could do that!

**Pull request merged**